test/compress/object.js

```diff
@@ -294,10 +297,13 @@ property_with_unprintable: {
             get "\x00\x01"() {
                 return "bar"
             }
+            set "\x00\x01"(foo) {
+                save(foo);
+            }
             *"\x00\x01"() {
                 return "foobar";
             }
         }
     }
-    expect_exact: 'var foo={"\\0\x01":"foo",get"\\0\x01"(){return"bar"},*"\\0\x01"(){return"foobar"}};class bar{get"\\0\x01"(){return"bar"}*"\\0\x01"(){return"foobar"}}'
+    expect_exact: 'var foo={"\\0\x01":"foo",get"\\0\x01"(){return"bar"},set"\\0\x01"(foo){save(foo)},*"\\0\x01"(){return"foobar"}};class bar{get"\\0\x01"(){return"bar"}set"\\0\x01"(foo){save(foo)}*"\\0\x01"(){return"foobar"}}'
 }
```

Comment:
Can you make a new test that's a copy of the test `property_with_unprintable` except using `ascii_only=true`?